### DISCUSSION TITLE
水印有点模糊

### DISCUSSION THREAD

**Discussion by Xm798** [Arondight/Adachi-BOT#590] [General]
![image](https://user-images.githubusercontent.com/48074861/150370703-924d4c1c-3990-411a-aa87-0a5348adb13b.png)
如图……倒也不是什么大问题就是

@Mark9804

**Comment by mark9804** [comment]
因为背景图片是糊的，已经不打算继续用下去了

米忽悠已经给我 <信息> 的参考答案了，我真傻
<img src="https://user-images.githubusercontent.com/9006264/150132787-7fdf80f8-89b3-440f-92a2-9cb07baf4a2d.jpeg" alt="CDA125AD-DA82-404A-8389-2904D437967C" width="200">

_Originally posted by @Mark9804 in https://github.com/Arondight/Adachi-BOT/issues/515#issuecomment-1016430160_

**Comment by Xm798** [comment]
> 因为背景图片是糊的，已经不打算继续用下去了
> 
> 
> 
> 米忽悠已经给我 <信息> 的参考答案了，我真傻
> 
> <img src="https://user-images.githubusercontent.com/9006264/150132787-7fdf80f8-89b3-440f-92a2-9cb07baf4a2d.jpeg" alt="CDA125AD-DA82-404A-8389-2904D437967C" width="200">
> 
> 
> 
> _Originally posted by @Mark9804 in https://github.com/Arondight/Adachi-BOT/issues/515#issuecomment-1016430160_

原来是背景图……2333
这个信息图好看诶

**Reply by mark9804** [reply]
> 原来是背景图……

历史遗留问题，我偶尔有救一下的冲动，但是每次最后都放弃了

重构是有极限的，所以我不重构辣
